google-cloud-storage/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+  <difference>
+    <differenceType>7012</differenceType>
+    <className>com/google/cloud/storage/Storage</className>
+    <method>void upload(*.BlobInfo, java.nio.file.Path, *.Storage$BlobWriteOption[])</method>
```

Comment:
An incompatible change like this requires a major version update since this API post 1.0

Reply:
Yes, I understand, what is the procedure for such kind of changes?

Reply:
I think at a minimum you'd need to update the version in the root pom.xml in this PR.